src/Workspaces/Core/Portable/Rename/RenameLocation.ReferenceProcessing.cs

```diff
@@ -279,8 +279,9 @@ private static string TrimNameToAfterLastDot(string name)
                 // destructors declarations that match the name
                 if (referencedSymbol.Kind == SymbolKind.NamedType)
                 {
-                    var namedType = (INamedTypeSymbol)referencedSymbol;
+                    ISyntaxFactsService syntaxFacts = null;
```

Comment:
yes.  it just looks ugly as sin.  You're going to get the locations in 99.999% of all cases.  So just get the value up front so you can use it :)